squish/src/lib.rs

```diff
@@ -333,4 +333,59 @@ mod tests {
         let estimate = Format::Bc3.compressed_size(15, 30);
         assert_eq!(estimate, 512);
     }
+
+    // The test-pattern is a gray-scale checkerboard of size 4x4 with 0xFF in the top-left.
+    // On top of that, the four middle pixels are set to 0x7F.
+    static DECODED_BLOCK_GRAY_4X4: &[u8] = &[
+        0xFF, 0x00, 0xFF, 0x00, // row 0
+        0x00, 0x7F, 0x7F, 0xFF, // row 1
+        0xFF, 0x7F, 0x7F, 0x00, // row 2
+        0x00, 0xFF, 0x00, 0xFF, // row 3
+    ];
```

Comment:
Same thing as in the other PR, I'm not a huge fan of having test data at the top of the tests namespace, and the variable name could describe the pattern or issue it is testing.